viewer/viewer.js

```diff
@@ -147,6 +148,15 @@ app.locals.molochClusters = Config.configMap("moloch-clusters");
 app.use(favicon(__dirname + '/public/favicon.ico'));
 app.use(passport.initialize());
 
+app.use(helmet.hidePoweredBy());
+app.use(helmet.frameguard({ action: 'deny' }));
+app.use(helmet.xssFilter());
+app.use(helmet.hsts({
```

Comment:
personally I would rather have the if outside to save it running everytime, but maybe that is the C programmer in me.  
if (Config.get('hstsHeader', false) && Config.isHTTPS()) {
  app.use(helmet ...
}